packages/apollos-data-prayer/src/prayer-requests/__tests__/resolver.test.js

```diff
@@ -337,13 +359,15 @@ describe('PrayerRequest', () => {
   it('creates a new prayer', async () => {
     const query = `
       mutation {
-        addPublicPrayerRequest(FirstName: "Test", LastName: "Bro", Text: "Jesus Rocks", CampusId: 16, CategoryId: 1, IsAnonymous: "True") {
+        addPublicPrayerRequest(FirstName: "Test", LastName: "Bro", Text: "Jesus Rocks", CampusId: Campus:772fcb6087247ebad630814e2ce0cd16, CategoryId: 1, IsAnonymous: true) {
```

Comment:
when this is PR'd against core we'll need to convert these arguments from `PascalCase` to `camelCase`, and if `CategoryId` is a Rock ID, it needs to be changed to an Apollos ID.

Reply:
Yes on the camel case. Not sure what you mean about `CategoryId` but I do find it odd that it's the only variable we are hardcoding to 2 everywhere.

Reply:
the value `2` is probably directly tied to something in Rock. If so, we need to abstract it away from Rock into something that makes sense in the context of Apollos. If the `categoryId` is `2` everywhere, it should probably just be removed from the mutation.

Reply:
we use `parseGlobalId` and such for `id`s elsewhere...that's what I mean.

Reply:
yep agreed